### PR TITLE
wrong link is replaced by correct link#5436

### PR DESCRIPTION
**Pull request by RAJU2529**
as per user issue # 5436, I have replaced the wrong link to new link
wrong link
https://msdn.microsoft.com/library/windows/hardware/aa394505

new link
https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-videocontroller

also, i added the related link for reference

### CODE REVIEW

File: windows/client-management/mdm/wmi-providers-supported-in-windows.md [hunk 1]
```diff
@@ -296,15 +296,16 @@ For links to these classes, see [**MDM Bridge WMI Provider**](https://msdn.micro
...
 **Win32\_WindowsUpdateAgentVersion**                                                        |
  
 
 ## Related topics
 
-
 [Configuration service provider reference](configuration-service-provider-reference.md)
 
+## Related Links
+[CIM Videocontroller] (https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/cim-videocontroller)
```
Comment (illfated) [inline windows/client-management/mdm/wmi-providers-supported-in-windows.md]:
```suggestion
[CIM Videocontroller](https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/cim-videocontroller)
```
(Link text and URL in parentheses must be next to each other without space between them, in order to appear as one item. They will appear separately, as seen in this source, if there is a space between them.)

File: windows/client-management/mdm/wmi-providers-supported-in-windows.md [hunk 2]
```diff
@@ -296,15 +296,16 @@ For links to these classes, see [**MDM Bridge WMI Provider**](https://msdn.micro
...
 **Win32\_WindowsUpdateAgentVersion**                                                        |
  
 
 ## Related topics
 
-
 [Configuration service provider reference](configuration-service-provider-reference.md)
 
+## Related Links
+[CIM Videocontroller](https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/cim-videocontroller)
```
Comment (illfated) [inline windows/client-management/mdm/wmi-providers-supported-in-windows.md]: Compared to other languages, English uses very few combined words. We should use either the technical form [CIM_VideoController] or the free text form **CIM Video Controller** (title format).
```suggestion
[CIM Video Controller](https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/cim-videocontroller)
```

File: windows/client-management/mdm/wmi-providers-supported-in-windows.md [hunk 3]
```diff
@@ -296,15 +296,16 @@ For links to these classes, see [**MDM Bridge WMI Provider**](https://msdn.micro
 [**Win32\_UninterruptiblePowerSupply**](https://msdn.microsoft.com/library/windows/hardware/aa394503) |
 [**Win32\_USBController**](https://msdn.microsoft.com/library/windows/hardware/aa394504)    |
 [**Win32\_UTCTime**](https://msdn.microsoft.com/library/windows/hardware/aa394510)          | ![cross mark](images/checkmark.png)
-[**Win32\_VideoController**](https://msdn.microsoft.com/library/windows/hardware/aa394505) |
+[**Win32\_VideoController**](https://docs.microsoft.com/en-us/windows/win32/cimwin32prov/win32-videocontroller) |
```
Comment (ManikaDhiman) [inline windows/client-management/mdm/wmi-providers-supported-in-windows.md]: Links to pages on docs.microsoft.com sites are coded as locale agnostic, meaning that the URL does not include a locale, such as en-us, en-gb, en-in, or any other locale. Please remove "en-us" from the URL. Thanks!

Comment (illfated) [reply windows/client-management/mdm/wmi-providers-supported-in-windows.md]: Fair point. Tested the link using only
https://docs.microsoft.com/windows/win32/cimwin32prov/win32-videocontroller
and it works just fine.

Comment (Dansimp) [reply windows/client-management/mdm/wmi-providers-supported-in-windows.md]: removed en-us from the URL

**Comment by illfated** [comment]
Ref. #5436

**Pull request merged**